src/lib/components/EventScheduleModal/index.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BaseModalElement} from '../BaseModalElement';
+import {html} from 'lit-element';
+import './_styles.scss';
+import {generateIdSalt} from '../../utils/generate-salt';
+
+/**
+ * @fileoverview Modal element for session information.
+ */
+
+class EventScheduleModal extends BaseModalElement {
+  static get properties() {
+    return {
+      sessionRow: Element,
+      _sessionName: String,
+      _authorsPart: Element,
+      _abstractPart: Element,
+      _titleId: String,
+    };
+  }
+
+  constructor() {
+    super();
+
+    this._titleId = generateIdSalt();
+  }
+
+  shouldUpdate(changedProperties) {
+    if (!changedProperties.has('sessionRow')) {
+      return false;
+    }
+
+    this._authorsPart = null;
+    this._abstractPart = null;
+    this._sessionName = '';
+
+    if (!this.sessionRow) {
+      return true;
+    }
+    const row = this.sessionRow;
+
+    // Extract the authors' collection and abstract.
+    this._authorsPart = row.querySelector('.w-event-schedule__speaker');
+    this._abstractPart = row.querySelector('.w-event-schedule__abstract');
+    if (this._abstractPart) {
+      this._abstractPart.removeAttribute('hidden');
+    }
+
+    // Remove the link that was used to open us.
+    const link = row.querySelector('.w-event-schedule__open');
+    if (link) {
+      this._sessionName = link.textContent;
+    }
+    return true;
+  }
+
+  render() {
+    return html`
```

Comment:
Should have "About this session" in the top left corner.

![image](https://user-images.githubusercontent.com/1066253/84688175-c09caa00-aef3-11ea-9cef-789ae1cef09f.png)